tests/testthat/test-lookup_users.R

```diff
@@ -0,0 +1,10 @@
+test_that("lookup_users works", {
+  users <- c(
+    "potus", "hillaryclinton", "realdonaldtrump",
+    "fivethirtyeight", "cnn", "espn", "twitter"
+  )
+  usr_df <- lookup_users(users)
+  expect_s3_class(usr_df, "tbl")
+  expect_equal(nrow(usr_df), 6)
+  expect_equal(ncol(usr_df), 91)
```

Comment:
I think it's risky to hard code these numbers because they might fail for reasons outside of our control (i.e. a user gets banned, or the Twitter api changes what fields it returns).

Reply:
Precisely, I would like to know if the API changes or the user we are using for testing and the examples is banned (we could always use the user specific for rtweet to avoid last problem).

Reply:
Ok, makes sense. The main thing is to acknowledge and accept the risk.